iphone/Classes/TiUIWindowProxy.m

```diff
@@ -956,6 +955,7 @@ - (void)setupWindowDecorations
   SETPROP(@"translucent", setTranslucent);
   SETPROP(@"tabBarHidden", setTabBarHidden);
   SETPROPOBJ(@"toolbar", setToolbar);
+  [[controller navigationController] setToolbarHidden:!hasToolbar animated:YES];
```

Comment:
Seriously? Wow.  Can we still ensure this doesn't cause side-effects for other use-cases like changing title-attributes etc?